src/K9OCRS/ClientApp/src/areas/accounts/modules/sagas.js

```diff
@@ -27,6 +27,7 @@ function* logout() {
   try {
     log('Logging out');
     // Call the api endpoint that removes the http-only cookie for the logged in user
+    const response = yield call(accountsApi.logout);
```

Comment:
Here you can use `yield call(accountsApi.logout);` without assigning its result to `const response` since you're not using the response later on. This causes a linting warning `'response' is assigned a value but never used`

You can just apply this suggestion and it'll do the trick. :+1:

```suggestion
    yield call(accountsApi.logout);
```